### PR TITLE
1618: branches blacklist

### PR DESCRIPTION
**Pull request by xendk**
Fixes up #480

### CODE REVIEW

**Comment by xendk** [comment]
Scrutinizer har drukket af natpotten. Den indentering den kræver er hul i hovedet.

File: modules/fbs/fbs.module [hunk 1]
```diff
@@ -575,3 +620,33 @@ function _fbs_wayf_login($name, $value = NULL) {
...
+  }
+}
+
+/**
+ * Quote string for use in CQL.
+ *
+ * Adds " around the string and escapes inline ".
+ */
+function _fbs_cql_quote_string($name) {
+  return '"' . preg_replace('/"/', '//"', $name) . '"';
```
Comment (holt83) [inline modules/fbs/fbs.module]: When testing it seems like only \\" is working for escaping. So shouldn't this be something like:
`preg_replace('/"/', '\\\\"', $name)`

Comment (xendk) [inline modules/fbs/fbs.module]: No, two is enough, we need to escape the backslash to make it part of the string. If it was in the regexp, four is needed to escape it in the string and escape it in the regexp, but as this is the replacement string, we don't need that.

But it should be backslashes, not slashes, yes.

Comment (holt83) [inline modules/fbs/fbs.module]: I see :)

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**